index.html

```diff
@@ -20,6 +20,28 @@ <h2>South Korea</h2>
       <img src="images/temple_home.jpg" alt="picture of temple">
       <img src="images/waterfall_home.jpg" alt="picture of waterfall">
     </div>
+    <div class="content">
+      <nav>
+        <h3>Beautiful Cities</h3>
+        <p>Seoul is am amazing city, that has so miuch to offer. Great <a href="places_to_eat.html">food</a>, amazing people, and tons of sights to see! </p>
+      </nav>
+
+      <nav>
+        <h3>Breathtaking Nature</h3>
+        <p>South Korea is one of the most beautful places in the world! There are stunning mountains, amazing hiking trails, stunning beaches, and so much more to be seen!</p>
+      </nav>
+
+      <nav>
```

Comment:
Instead of using the nav tag under content, you can use a div and just give it a class to manipulate those sections. Nav or navigation tag is used usually for the top menu and to help the user navigate the site.